README.md

```diff
@@ -74,6 +74,18 @@ use {"akinsho/toggleterm.nvim", tag = '*', config = function()
 end}
 ```
 
+Using `lazy.nvim` in lua
+
+```lua
+require("lazy").setup({
```

Comment:
Also I'm not sure if showing the user that they have to setup lazy is a good idea I'm not sure if a new user will assume that they need a completely separate setup call just to setup this plugin

Reply:
I was also debating this, but checked how other plugins handle it in their READMEs and decided to stick with this version after seeing the references.